core/libraries/form_sections/strategies/validation/EE_Equal_To_Validation_Strategy.strategy.php

```diff
@@ -0,0 +1,61 @@
+<?php
+use EventEspresso\core\domain\services\factories\EmailAddressFactory;
+use EventEspresso\core\domain\services\validation\email\EmailValidationException;
+use EventEspresso\core\exceptions\InvalidDataTypeException;
+use EventEspresso\core\exceptions\InvalidInterfaceException;
+
+/**
+ * Class EE_Email_Validation_Strategy
+ *
+ * @package               Event Espresso
+ * @subpackage            core
+ * @author                Mike Nelson
+ * @since                 4.6
```

Comment:
# AHA!!!

I had my suspicions that you were actually Michael Nelson in disguise!!!

Nah just being silly, but you should give yourself credit for this code even if it is modelled after code that Michael wrote.
As well, for any `@since` tags for new code, we use `$VID:$` for the version number which will get automagically replaced by the correct version when a release is built.

```suggestion
 * @author                Rafael Goulart
 * @since                 $VID:$
```